src/main/java/me/totalfreedom/totalfreedommod/config/ConfigEntry.java

```diff
@@ -49,6 +49,7 @@
     //
     LANDMINES_ENABLED(Boolean.class, "landmines_enabled"),
     TOSSMOB_ENABLED(Boolean.class, "tossmob_enabled"),
+    MAKE_ARROW_EXPLOSIVES(Boolean.class, "arrow_explosive_enabled"),
```

Comment:
why are you setting somethings "explosives" and then "explosive"?

Reply:
Ignore that, no one will read on tf code..

Reply:
🤔